docs/frontend-system/architecture/60-migrations.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: migrations
-title: Frontend System Migrations
-sidebar_label: Migrations
+title: Frontend System Changelog
```

Comment:
This is where you land if you search for frontend migration, which is not what we want

Reply:
...but the id is still `migrations`? Alright